packages/react-native/React/Views/ScrollView/RCTScrollView.m

```diff
@@ -324,7 +325,20 @@ - (void)_keyboardWillChangeFrame:(NSNotification *)notification
   }
 
   CGPoint newContentOffset = _scrollView.contentOffset;
-  CGFloat contentDiff = endFrame.origin.y - beginFrame.origin.y;
+  self.firstResponderFocus = CGRectNull;
+
+  CGFloat contentDiff = 0;
+  if ([[UIApplication sharedApplication] sendAction:@selector(reactUpdateResponderOffsetForScrollView:) to:nil from:self forEvent:nil]) {
+    // Inner text field focused
+    CGFloat focusEnd = self.firstResponderFocus.origin.y + self.firstResponderFocus.size.height;
+    if (focusEnd > endFrame.origin.y && focusEnd != INFINITY) {
```

Comment:
When can you ever get `INFINITY` here?

Reply:
@javache It happens when the focussed text view is not a child of the scroll view. I've updated the code to make that clearer.